app/javascript/components/EditInterview.jsx

```diff
@@ -0,0 +1,112 @@
+import axios from 'axios'
+import {NavLink} from 'react-router-dom'
+import InterviewForm from './InterviewForm'
+import React, {useState, useEffect} from 'react'
+import  { Redirect } from 'react-router-dom'
+
+
+function EditInterview(props) {
+   
+   
+    let [title, setTitle] = useState(props.location.state.interview.title);
```

Comment:
1. Its better to use const here. You don't want to override values returned from hook
2. Use object destructuring instead of chaining

Reply:
commit - 832864a